exchanges/gateio/gateio.go

```diff
@@ -2849,7 +2850,8 @@ func (g *Gateio) PlaceDeliveryOrder(ctx context.Context, arg *OrderCreateParams)
 	if arg.Size == 0 {
 		return nil, fmt.Errorf("%w, specify positive number to make a bid, and negative number to ask", errInvalidOrderSide)
 	}
-	if arg.TimeInForce != gtcTIF && arg.TimeInForce != iocTIF && arg.TimeInForce != pocTIF && arg.TimeInForce != fokTIF {
+	arg.TimeInForce = strings.ToLower(arg.TimeInForce)
+	if !slices.Contains([]string{gtcTIF, iocTIF, pocTIF, fokTIF}, arg.TimeInForce) {
```

Comment:
Can you just check `[]string{gtcTIF, iocTIF, pocTIF, fokTIF}` it might allocate on every request so could global this?